test/libsolidity/syntaxTests/unimplemented_super_function.sol

```diff
@@ -5,4 +5,4 @@ contract b is a {
     function f() public override { super.f(); }
 }
 // ----
-// TypeError: (110-117): Member "f" not found or not visible after argument-dependent lookup in contract super b.
+// TypeError: (110-119): Call to unimplemented function "f".
```

Comment:
I don't think this should be an error in general. At this point, we cannot tell whether there is a super function or not.

Reply:
Really? I mean, the check is very specifically checking for this situation? Can you think of a scenario where this message is wrong?

Reply:
`super` performs virtual function lookup. This means the lookup is performed differently depending on the final derived contract that is being compiled. In the following example, `super.f()` should compile for `D`.

```
contract A { function f() public pure virtual; }
contract B is A { function f() public pure override virtual { super.f(); } }
contract C is A { function f() public pure override virtual { } }
contract D is B, C { function f() public pure virtual override(B, C) { super.f(); } }
```

I think we need to discuss what to do if `B` is to be compiled on its own. I think @ekpyron is of the opinion to remove `super` altogether.

Reply:
what does the `super` in contract `B` refer to then? (that's where it would throw an error with this PR)

Reply:
I suppose I can either exclude `super` from the check, or I set a tag in the annotation and only trigger errors when the final contract was processed... though that could get messy

Reply:
Are you sure about your example? I think this doesn't even work with the current `develop` version

Reply:
Please only fix the non-super case. I said that `super.f()` should compile. It is a bug that it currently does not compile, but it is a different bug that #7314